internal/lockservice/lockservice.go

```diff
@@ -23,4 +23,5 @@ type LockService interface {
 // Descriptors describe the type of data that a lock acquiring component must describe.
 type Descriptors interface {
 	ID() string
+	Owner() string
```

Comment:
If this is done, then every item in the lock map would be unique right? We had discussed that the lock map will be `item:owner`, instead of `item:bool`. Is it possible to achieve this if FileID` and `OwnerID` are combined in `ID()`?